Tests/Adobe.Target.Client.Test/TargetClientConfigBuildShould.cs

```diff
@@ -30,6 +31,7 @@ public void Build_ReturnClientConfig()
         {
             var testWebProxy = new Mock<WebProxy>().Object;
             var testLogger = new Mock<ILogger>().Object;
```

Comment:
As far as I see this is not a very useful test, we are just checking that the config has the values we created it with

Reply:
We're verifying the config builder in this test